src/tiled/command.cpp

```diff
@@ -30,39 +30,70 @@
 using namespace Tiled;
 using namespace Tiled::Internal;
 
+QString Command::finalWorkingDirectory() const
+{
+    QString finalWorkingDirectory = workingDirectory;
+
+    finalWorkingDirectory = replaceVariables(finalWorkingDirectory, false);
+
+    QString finalExecutable = replaceVariables(executable);
+    QFileInfo mFile(finalExecutable);   //Check if executable is a path or not
+
+    if (mFile.exists() && mFile.isFile()) {
+        finalWorkingDirectory.replace(QLatin1String("%executablepath"),
+                            mFile.absolutePath());
```

Comment:
Coding style: please align wrapped function arguments (so `mFile.absolutePath());` should start at the same position as `QLatin1String` in the line above).